src/components/Extensions/Details/DetailCard/Tabs/EventsConfig.module.scss

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.wrapper {
+  --accordion-border-size: 0;
+  --accordion-horizontal-spacing: 0;
+  --accordion-vertical-spacing: 0;
+  --accordion-focus-background: transparent;
+  --accordion-header-justification: flex-start;
+
+  padding: 8px var(--tab-spacing);
+  margin-bottom: 16px;
+
+  p:first-child {
+    margin-top: 0;
+  }
+}
+
+.eventPrefixValue {
```

Comment:
nit: it seems like we're mixing snake-case and kebab... we should use 1, and my vote would be for kebab in css